addons/binding/org.openhab.binding.chamberlainmyq/src/main/java/org/openhab/binding/chamberlainmyq/config/ChamberlainMyQDeviceConfig.java

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.chamberlainmyq.config;
+
+import static org.openhab.binding.chamberlainmyq.ChamberlainMyQBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.openhab.binding.chamberlainmyq.internal.json.Attribute;
+import org.openhab.binding.chamberlainmyq.internal.json.Device;
+
+/**
+ * The {@link ChamberlainMyQDeviceConfig} class represents the common configuration
+ * parameters of a MyQ Device
+ *
+ * @author Scott Hanson - Initial contribution
+ *
+ */
+public class ChamberlainMyQDeviceConfig {
+    private int deviceId;
+    private String deviceType;
+    private int deviceTypeId;
+    private int state;
+    private String description;
+    private String serialNumber;
+    private int parentId;
+    private boolean online;
+
+    public ChamberlainMyQDeviceConfig(Map<String, String> properties) {
+        this.deviceId = (int) Double.parseDouble(properties.get(MYQ_ID).toString());
```

Comment:
Why do you parse a `double` if you need an `int`, why do call `toString` on a `String`?